assets/js/modules/adsense/components/settings/SettingsView.js

```diff
@@ -134,6 +123,22 @@ export default function SettingsView() {
 					</p>
 				</div>
 			</div>
+
+			<div className="googlesitekit-settings-module__meta-items">
+				<div className="googlesitekit-settings-module__meta-item">
+					{ webStoriesActive && (
```

Comment:
This needs to wrap around the entire new `div` set here, since the only reason the extra `div`s are there is for the new setting control.